src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -5,9 +5,10 @@ import RequiredLoginView from '../../common/components/RequiredLoginView';
 import RequiredVeteranView from '../components/RequiredVeteranView';
 import EmailCapture from './EmailCapture';
 
-const rateLimitAuthed = window.settings.vic.rateLimitAuthed;
-const rateLimitUnauthed = window.settings.vic.rateLimitUnauthed;
-const serviceRateLimitedRandomizer = Math.random();
+const vicSettings = {
+  serviceRateLimitedUnauthed: Math.random() > window.settings.vic.rateLimitUnauthed,
+  serviceRateLimitedAuthed: window.sessionStorage.getItem('vicInitialAuthStatus') !== 'unauthenticated' && Math.random() > window.settings.vic.rateLimitAuthed
```

Comment:
I'd add a comment here that says, users who have made it past the unauthenticated rate-limiting gate, and are now authenticated and accessing VIC should not be subject to the authenticated rate-limiting gate

Or something much more succinct! Just think the double negative above, `!== 'unauthenticated'` makes this hard to grok and I want to make sure the intent is clear.

Reply:
Yeah good idea, I'll do that and wrap that logic in a function so I can list where the variables come from and add some meaning to them.